rest/client/methods.go

```diff
@@ -525,6 +529,40 @@ func (c *communicatorImpl) SendTestLog(ctx context.Context, taskData TaskData, l
 		return "", errors.New(message)
 	}
 	logID := logReply.ID
+
+	// Send to buildlogger.
+	tk, err := c.GetTask(ctx, taskData)
```

Comment:
Should sending test logs to Cedar be done external to the communicator request? If we're going to gate sending test logs/results to cedar by a project setting, it seems like it would be easier to do this logic optionally after the places that call `SendTestLogs`. I purposely commented out sending test results to cedar unti the project setting exists.

Reply:
I made it external because we the Communicator already holds a grpc connection to cedar.

Reply:
Sure, but I was thinking that it would be weird later because we'll have to change `SendTestLog` to pass in the project setting for whether that project should send test logs to Cedar. I was thinking it should be outside of this function and you can just call `GetCedarGRPCConn` like [`sendTestResultsToCedar`](https://github.com/evergreen-ci/evergreen/blob/e1a1a040cb99960a0422e85928d280f08e8cb897/command/results_utils.go#L86) does.